ee/server/service/software_installers.go

```diff
@@ -389,6 +392,9 @@ func (svc *Service) UpdateSoftwareInstaller(ctx context.Context, payload *fleet.
 		if uninstallScript == "" { // extension can't change on an edit so we can generate off of the existing file
 			uninstallScript = file.GetUninstallScript(existingInstaller.Extension)
 		}
+		if uninstallScript == "" {
+			return nil, ctxerr.New(ctx, fmt.Sprintf("uninstall script must be provided for %s packages", strings.ToUpper(existingInstaller.Extension)))
```

Comment:
@iansltx I think we want the "Couldn't add. Install script is required for .exe packages":

<img width="864" alt="Screenshot 2025-03-19 at 11 31 45 AM" src="https://github.com/user-attachments/assets/89ca0107-28ae-4be1-8ea4-44cfce047e3f" />

Same tweak for uninstall error.